src/components/inputs/checkbox-input/checkbox-input.styles.js

```diff
@@ -30,11 +38,11 @@ const getCheckboxWrapperStyles = props => {
       baseStyles,
       css`
         svg [id$='borderAndContent'] > [id$='content'] {
-          fill: ${vars.borderColorInputError};
+          fill: ${overwrittenVars[designTokens.borderColorForInputWhenError]};
```

Comment:
do we want borderColor for the fill here ?

Reply:
Yeah it might make more sense semantically to use the design token fontColorForInputWhenError because here are targeting the id "content", which is the check mark (when checked and error). But that token is not red 🤔. It might make more sense just to use `colorError`

It's pretty confusing, I wanted to rewrite this with CSS and get rid of the SVG usage but haven't got around to it.